### PR TITLE
Connect match reqs

### PR DESCRIPTION
**Pull request by louonezime**
# Description

Please provide a detailed description of what was done in this PR.
Precise the issue that you are resolving.

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Breaking changes

Please complete this section if any breaking changes have been made, otherwise delete it.

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have added the needed labels
- [x] I have linked this PR to an issue
- [x] I have linked this PR to a milestone
- [ ] I have linked this PR to a project
- [x] I have tested this code
- [ ] I have added / updated tests (unit / functional / end-to-end / ...)
- [ ] I have updated the README and other relevant documents (guides...)
- [ ] I have added sufficient documentation both in code, as well as in the READMEs

# Additional comments

Please post additional comments in this section if you have them, otherwise delete it.

### CODE REVIEW

**Comment by louonezime** [comment]
delete ton dossier cmd @57ave

File: src/cli/cli.go [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+		fmt.Fprintf(os.Stderr, "Usage: bruteforce [options] <url>\n")
+		fmt.Fprintf(os.Stderr, "Options:\n")
+		flag.PrintDefaults()
+	}
+
+	flag.Parse()
+
+	if params.Workers < 1 {
+		return params, ThreadsError
+	}
```
Comment (pierrelissope) [inline src/cli/cli.go:35]: Add a worker limit

File: cmd/callWorker.go [hunk 2]
```diff
@@ -0,0 +1,40 @@
+package main
```
Comment (louonezime) [inline cmd/callWorker.go:1]: rm -rf

File: src/cli/cli.go [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+import (
+	"bruteforce/src/models"
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+)
+
+func Parse_cli_args() (models.Forcing_params, error) {
```
Comment (louonezime) [inline src/cli/cli.go:11]: snakecase?

File: src/main.go [hunk 4]
```diff
@@ -1,9 +1,19 @@
...
 import (
+	"bruteforce/src/cli"
+	"bruteforce/src/query"
 	"fmt"
 )
 
 func main() {
-	fmt.Println("Hello World")
+
+	forcing_params, err := cli.Parse_cli_args()
```
Comment (pierrelissope) [inline src/main.go:11]: camel case stp

**Pull request merged**